src/Database/Statement/StatementDecorator.php

```diff
@@ -210,11 +263,21 @@ public function fetch($type = 'num')
      * @param string $type num for fetching columns as positional keys or assoc for column names as keys
      * @return array List of all results from database for this statement
      */
-    public function fetchAll($type = 'num')
+    public function fetchAll($type = self::FETCH_TYPE_NUM)
     {
         return $this->_statement->fetchAll($type);
     }
 
+    /**
+     * Returns an array of \stdClass objects or false
+     *
+     * @return \stdClass[]|false
+     */
+    public function fetchAllObjects()
```

Comment:
Why does `fetchAllObjects()` require a method? The other 'all' types don't have their own methods.

Reply:
Happy to add them.

Reply:
Or remove this one? We usually lean towards exposing fewer methods to start with. We can always add more later, but taking methods away takes a very long time.